COMPILING.md

```diff
@@ -482,3 +482,42 @@ successfully on Windows or macOS.
    The argument for the IPASIR parameter gives the build system the location for
    the IPASIR headers, which is needed for the cbmc includes of `ipasir.h`. The
    compiled binary will be placed in `cbmc/src/cbmc/cbmc`.
+This document assumes you have already been able to build CPROVER on
+your chosen architecture. For details on building CPROVER see
+COMPILING.md.
+
+#RUNNING REGRESSION TESTS
+
+Regression tests can be run using cmake or make. Your choice here
+should be the same as the compiling of the project itself.
+
+Note that running all regression tests can be slow when a debug
+build of CPROVER is used.
+
+## CMAKE
+
+This can be done by changing to the directory you built the
+project in with cmake and running ctest as follows.
+```
+cd <build_dir>
+ctest . -V -L CORE
+```
+You can also specify a pattern of tests to run as follows.
+```
+ctest . -V -L CORE -R <pattern>
+```
+For example
+```
+ctest . -V -L CORE -R goto
+```
+that will run all CORE tests that include `goto` in their
+name.
+
+## MAKE
+
+This can be done by changing to the regression directory and
+simply running make as follows.
+```
+cd regression
+make
```

Comment:
I believe there's a slight difference between cmake/make in what tests actually run, which might be worth either clarifying, or adjusting the instructions to elide. That is around unit tests... I believe the `ctest` commands run _both_ regression _and_ unit tests, whereas I though the make command you show here _only_ runs the regression tests. My recommendation might be to tweak the section title to say "RUNNING REGRESSION AND UNIT TESTS", and then tweak the instructions for make to show how to run unit tests as well?

Reply:
I think this is clearer now.